app/uk/gov/hmrc/gform/views/form/snippets/field_template_address.scala.html

```diff
@@ -48,13 +49,13 @@
 <fieldset class="govuk-fieldset" id="@{fieldValue.id.value}-fieldset" role="group" @if(fieldValue.helpText.nonEmpty) { aria-describedby="@{fieldValue.id.value}-hint" }>
 
     @if(formLevelHeading) {
-        <legend class="govuk-fieldset__legend govuk-label--l">
+        <legend class="govuk-fieldset__legend govuk-label--l @classes">
```

Comment:
Why do we need @classes if its formLevelHeading? it should be the default page level heading size i.e govuk-label--l right